### PR TITLE
Disable interactions

### PR DESCRIPTION
**Pull request by joaompneves**
I know that this is alot of diffs, but i took the opportunity to cleanup and organize the code. sorry
just give a look even if you dont understand it all. tks

### CODE REVIEW

File: ReactViewControl/ReactViewRender.LoaderModule.cs [hunk 1]
```diff
@@ -137,13 +132,13 @@ private class SyncFunction {
                 ViewRender.WebView.ExecuteScript($"import('{loaderUrl}').then(m => m.default.{LoaderModuleName}).then({LoaderModuleName} => {LoaderModuleName}.{functionName}({string.Join(",", args)}))");
             }
 
-            private static string SerializeComponent(IViewModule component, bool forceNativeSyncCalls) {
+            private static string SerializeComponent(IViewModule component) {
                 var nativeObjectMethodsMap = component.Events
-                    .Select(g => new KeyValuePair<string, object>(g, forceNativeSyncCalls ? SyncFunction.Instance : JavascriptSerializer.Undefined))
+                    .Select(g => new KeyValuePair<string, object>(g, JavascriptSerializer.Undefined))
```
Comment (Jmales) [inline ReactViewControl/ReactViewRender.LoaderModule.cs:137]: Why don't we need this "forceNativeSyncCalls" anymore?

Comment (joaompneves) [reply ReactViewControl/ReactViewRender.LoaderModule.cs:137]: that mechanism had problems and I removed

**Review by Jmales — APPROVED** [review]

File: ReactViewControl/FrameInfo.cs [hunk 2]
```diff
@@ -5,6 +5,8 @@ namespace ReactViewControl {
 
     internal class FrameInfo : IFrame {
 
+        internal const string MainViewFrameName = "";
```
Comment (alvesmiguel1) [inline ReactViewControl/FrameInfo.cs:8]: string.Empty

Comment (joaompneves) [reply ReactViewControl/FrameInfo.cs:8]: string.empty is a static.. this is a const

Comment (alvesmiguel1) [reply ReactViewControl/FrameInfo.cs:8]: Oh, didn't notice it

**Pull request merged**